lists/AT-ZVR

```diff
@@ -0,0 +1,55 @@
+{
+  "name": {
+    "en": " Austrian Central Register of Associations (AT-ZVR)",
+    "local": "Zentrales Vereinsregister"
+  },
+  "url": "https://www.bmi.gv.at/609/zvr.aspx",
+  "coverage": [
+    "AT"
+  ],
+  "description": {
+    "en": "The central register of associations is the database of registered associations in Austria, identified by their official registration number. Anyone can use the Central Register of Associations (ZVR) to search the register for any association that can be clearly identified by its ZVR number, or its name, or part of its name and can request information about the data of a clearly identifiable association for which there is no information block (extract from the association register)."
+  },
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "trust"
+  ],
+  "sector": [],
+  "code": "AT-ZVR",
+  "deprecated": false,
+  "confirmed": true,
+  "listType": "primary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://citizen.bmi.gv.at/at.gv.bmi.fnsweb-p/zvn/public/Registerauszug",
+    "guidanceOnLocatingIds": "Use the search box in the footer of the page to enter company registration number. This is the ZVR number.",
```

Comment:
Suggested edit:

"Use the search boxes at the foot of the page to find an association's details. Search by registration number (ZVR-Zahl) or by supplying all or part of its name (Vereinsname). The latter type of search can be temperamental."